internal/forest/validator.go

```diff
@@ -0,0 +1,57 @@
+package forest
```

Comment:
Any reason not to put this in `config`? It doesn't seem to have anything to do with the rest of the `forest` package, but it does call into `config`.